src/main/java/com/aperturelabs/app/simulator/Adder.java

```diff
@@ -4,6 +4,8 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import com.aperturelabs.app.tsgui.Register;
```

Comment:
This doesn't need to be imported

Reply:
the dark deed you requested has been done